tests/moduleTests/vfShow.ts

```diff
@@ -72,6 +72,43 @@ export default function tests() {
         assert.deepEqual(renderer.vfTies[0].last_note, n2.activeVexflowNote);
     });
 
+    test('music21.vfShow.Renderer prepareTies in voices across barline', assert => {
+        const m1_sop_voice = new music21.stream.Voice();
+        const n_m1_v1 = new music21.note.Note('C5');
+        m1_sop_voice.repeatAppend(n_m1_v1, 2);
+        const m1_alto_voice = new music21.stream.Voice();
+        const n_m1_v2 = new music21.note.Note('E4');
+        m1_alto_voice.repeatAppend(n_m1_v2, 2);
+        const m1 = new music21.stream.Measure();
+        m1.append(m1_sop_voice);
+        m1.append(m1_alto_voice);
+
+        const m2_sop_voice = new music21.stream.Voice();
```

Comment:
why would this have the same v.id as the previous m1_sop_voice?  I don't see .id defined anywhere here.

`.id` is probably not the best use since it should be unique within a hierarchy, but I think that's what we have used.

Reply:
I guess two test cases would be beneficial. This is testing the default ordering of voices in the container. I was focusing on ensuring I had a test that failed on current version and forget to set id's -- You're right we need to test setting id, also.

Reply:
Done

Reply:
> .id is probably not the best use since it should be unique within a hierarchy, but I think that's what we have used.

Yeah, I think in https://github.com/cuthbertLab/music21/pull/1003#discussion_r627352719 we were discussing documenting that uniqueness is only at the first stream container, not across any sub-level of the hierarchy.

Reply:
> This is testing the default ordering of voices in the container.

Update: I ended up removing the regression test for default ordering of voices in the container. I thought it was happening to work correctly for sopranos only (not altos), but no... didn't work at all before...

Reply:
I think .id should be encouraged at all levels within a hierarchy and that something else (`.voiceId` or a group) should be used for this -- EVENTUALLY -- for now this is good.